packages/block-editor/src/hooks/border.js

```diff
@@ -16,7 +16,41 @@ import { BorderRadiusEdit } from './border-radius';
 import { BorderStyleEdit } from './border-style';
 import { BorderWidthEdit } from './border-width';
 
+const isWeb = Platform.OS === 'web';
+
 export const BORDER_SUPPORT_KEY = '__experimentalBorder';
+export const CSS_UNITS = [
```

Comment:
Not for this PR, but wanted to share some thoughts about this, in case someone here has bandwidth to act on them. I find asking the consumers of the `UnitControl` to pass a list of unit definitions creates a couple of issues:

- strings to translate: asking consumers of the `UnitControl` to redefine units (they're already defined here) creates the same multiple entries for folk to translate, which are the same.
- maintanibility: all consumers need to be changed when we want to change a label, for example.

It'd be better if the component only asked consumers to pass the allowed units and its defaults.

---

For this PR: I see that the units defined [here](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/unit-control/utils.js#L17-L17) also have a `a11ylabel`, which this lists doesn't have. Do we need it?

Reply:
I've added the `a11yLabel` properties to the `CSS_UNITS` config used by border support. 

It appears they are only used by the native UnitControl at the moment and they get retrieved from the UnitControl's `ALL_CSS_UNITS` config in `utils.js`. I still added them here for consistency and to withstand any changes that might happen to the UnitControl before your suggestion above can be implemented. I figure it being consistent will make that a smoother process as well.

On the topic of updating the UnitControl to only ask consumers to pass allowed units and defaults, that sounds good to me. I'll make a note of this in my backlog and try and free up some bandwidth to address it.

Reply:
I've created an issue for the proposed UnitControl changes. https://github.com/WordPress/gutenberg/issues/32478

Reply:
Turns out this has already been taken care of https://github.com/WordPress/gutenberg/issues/32478#issuecomment-855705065.

There's a PR up to update the use of `CSS_UNITS` here and in the letter-spacing hook (https://github.com/WordPress/gutenberg/pull/32482)